### PR TITLE
Task list edited

### PR DESCRIPTION
**Pull request by kaushiro**
At the moment I have the task and list showing under its respective topic and have limited it to the first two topics. Can you take a look at it and see if there is a better formatting you would rather use?

### CODE REVIEW

**Comment by kaushiro** [comment]
Instead of listing tasks and assessments underneath the topics, only the topic is displayed with a link that takes you to its page.

File: src/components/UncompletedTasks.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from "react-router-dom";
+
+export default ({userProgress, uncompletedTopics }) => {
+	return (
+		<div className="topics panel">
+			<div className="topics-header">
+				<div className="topics-header-icon"><i className="fa fa-2x fa-file" aria-hidden="true"></i></div>
+			</div>
+			<p className="uncompleted-topic-description">These are the next 2 topics you have left to complete:</p>
```
Comment (KyeBuff) [inline src/components/UncompletedTasks.js]: Would probably re-word this? Something like 'we think you should complete these next 2 topics.' or something along those lines?

Comment (KyeBuff) [reply src/components/UncompletedTasks.js]: Like we said yesterday I think visually this could sit inside the same 'panel' as the Progress Bar in the welcome area.

File: src/containers/UncompletedTasks.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { connect } from "react-redux";
+import UncompletedTasks from "../components/UncompletedTasks";
+
+
+const mapStateToProps = state => {
+	let userProgress = state.get('userProgress');
+	let completedShortTitles = userProgress.map(el=>el.split('.')[0]); 
+	let topics = state.get('topics');
+
+	let tasks = topics.map(topic => topic.get('tasks').size);
```
Comment (KyeBuff) [inline src/containers/UncompletedTasks.js]: We can shorten this to 1 line and reduce the number of variables. So amalgamate lines 10-12 to the below:

```
let tasks = topics.map(topic => topic.get('tasks').size + topic.get('assessments').size);
```

File: src/components/UncompletedTasks.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+export default ({userProgress, uncompletedTopics }) => {
+	return (
+		<div>
+			<p className="uncompleted-topic-description">We suggest you complete the next two topics:</p>
+	  	
+	  		<div className='uncompleted-tasks-list'>
+		  		{uncompletedTopics.map((topic, i) => 
+		  			<div key={i} className='uncompleted-tasks-list-item'>
+			  			<Link to={ '/prep/topic/' + topic.get('id') }>
+			  				<h5>{topic.get('short_title')}</h5>
```
Comment (KyeBuff) [inline src/components/UncompletedTasks.js]: I'm not sure it makes sense for the h5 to live inside of the Link component?

File: src/containers/UncompletedTasks.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import { connect } from "react-redux";
```
Comment (KyeBuff) [inline src/containers/UncompletedTasks.js]: Can you please add the refactored code that I sent you please?
